internal/generator/vector/output/otlp/transform.go

```diff
@@ -64,6 +64,7 @@ r.body = {"stringValue": to_string!(get!(.,["_internal","message"]))}
 	APILogAttributes = `
 r.attributes = append(r.attributes,
   [
+    {"key": "level", "value": {"stringValue": .level}},
```

Comment:
This seems to be for filtering audit logs. I don't think we need "level" in those log entries.

```suggestion
```